test/WebJobs.Script.Tests/ScriptStartupTypeDiscovererTests.cs

```diff
@@ -346,6 +346,118 @@ void CopyToBin(string path)
             }
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public async Task GetExtensionsStartupTypes_LegacyBundles_UsesExtensionBundleBinaries(bool hasPrecompiledFunctions)
```

Comment:
Nit: Both tests setup looks the same. Try refactoring the tests to use a common setup helper method.